UI/window-basic-main-outputs.cpp

```diff
@@ -714,6 +748,12 @@ void SimpleOutput::UpdateRecordingSettings()
 
 	} else if (videoEncoder == SIMPLE_ENCODER_NVENC) {
 		UpdateRecordingSettings_nvenc(crf);
+
+#ifdef ENABLE_HEVC
+	} else if (videoEncoder == SIMPLE_ENCODER_NVENC_HEVC) {
+		UpdateRecordingSettings_nvenc_hevc(crf);
+
+#endif
```

Comment:
Move line break down, and move #endif up.

Reply:
Just delete the line break then.

Reply:
Deleted. Done.